### PR TITLE
Allow mixed type for fact name

### PR DESCRIPTION
**Pull request by maltebaer**
Hey @actived 👋

Nice package. I love how easy it is to integrate!

What do you think about not constraining the type of the fact name? At least in my app, I do not always want to logg associative arrays. For example:

```php
$bar = [1, 2, 3];
logs()->error('foo', [$bar]);
```

### CODE REVIEW

File: src/Handler/MicrosoftTeamsRecord.php [hunk 1]
```diff
@@ -216,13 +216,16 @@ public function setSections(array $record): self
     }
 
     /**
-     * @param string $name
+     * @param mixed $name
```
Comment (mimmi20) [inline src/Handler/MicrosoftTeamsRecord.php]: Please do not use `mixed`, but write all types you want to support like the interface `Stringable`. Arrays and objects without the `__toString` function are not allowed by the `strval` function.

**Review by mimmi20 — CHANGES_REQUESTED** [review]

File: src/Handler/MicrosoftTeamsRecord.php (hunk 1 above)
Comment (maltebaer) [reply src/Handler/MicrosoftTeamsRecord.php]: Sure. Now only `int` and `string` types are allowed.

File: src/Handler/MicrosoftTeamsRecord.php [hunk 2]
```diff
@@ -216,14 +216,14 @@ public function setSections(array $record): self
...
 
     /**
-     * @param mixed $name
+     * @param string|int $name
      * @param mixed $value
      * @param bool $isQuoted
      * @return array
      */
-    public function getFact($name, $value, bool $isQuoted = false): array
+    public function getFact(string|int $name, $value, bool $isQuoted = false): array
```
Comment (mimmi20) [inline src/Handler/MicrosoftTeamsRecord.php]: `string|int` is only allowed for PHP 8.1+ in the type definition.

**Review by mimmi20 — CHANGES_REQUESTED** [review]

File: src/Handler/MicrosoftTeamsRecord.php (hunk 2 above)
Comment (maltebaer) [reply src/Handler/MicrosoftTeamsRecord.php]: Okay, I removed the type declaration.

**Review by mimmi20 — APPROVED** [review]

**Pull request merged**